offer-manager-domain/src/main/groovy/life/qbic/business/persons/Person.groovy

```diff
@@ -140,9 +151,6 @@ class Person {
 
     void removeAffiliation(Affiliation affiliation) {
         if (affiliation in affiliations) {
-            if (affiliations.size() == 1) {
-                throw new IllegalArgumentException("Cannot remove the last remaining affiliation: $affiliation")
-            }
```

Comment:
Do we really want to remove this one yet? As there is no compensation action at the moment I would advise against it right now.